document/core/exec/instructions.rst

```diff
@@ -2027,7 +2027,7 @@ Entering an exception handler :math:`H`
 Exiting an exception handler
 ............................
 
-When the end of a :ref:`try <syntax-try>` instruction is reached without a jump, exception or trap, then the following steps are performed.
+When the end of a :ref:`try <syntax-try>` instruction is reached without a jump, exception, or trap, then the following steps are performed.
```

Comment:
I think in this particular place you do _not_ want to include exceptions. ;)

Reply:
Ooops, slip of the hand. Thanks for spotting it! :blush:

Reply:
@rossberg: on a second look, these are the instructions for exiting the try without an exception being thrown. If an exception is thrown then we are in the case "Throwing an exception ...", right?

Reply:
Right.